_posts/en/posts/2015-12-14-segregated-witness.md

```diff
@@ -6,7 +6,6 @@ id: en-segregated-witness
 name: segregated-witness
 title: Segregated Witness Video Presentation
 permalink: /en/2015/12/14/segregated-witness/
-author: sipa
 related: false
 tags: [scalability]
 version: 1
```

Comment:
Do these version numbers need to be bumped for a change like this?

Reply:
No, it shouldn't affect the translations, right?